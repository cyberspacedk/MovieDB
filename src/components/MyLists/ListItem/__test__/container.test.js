import React from 'react';
import { shallow } from 'enzyme';
import ListItemContainer from '../container';

describe('Container: ListItemContainer', () => {
  const props = {
    item: {
      id: 1,
    },
    deleteListRequest: jest.fn(),
    history: {
      push: jest.fn(),
    },
  };

  const wrapper = shallow(<ListItemContainer {...props} />);
  const container = wrapper.dive();

  it('Snapshot: should match', () => {
    expect(container).toMatchSnapshot();
  });
});
