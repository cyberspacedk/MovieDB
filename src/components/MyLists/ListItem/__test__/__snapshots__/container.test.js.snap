// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container: ListItemContainer Snapshot: should match 1`] = `
<Col
  className="my-list-card"
  lg={
    Object {
      "span": 4,
    }
  }
  mg={
    Object {
      "span": 6,
    }
  }
  sm={
    Object {
      "span": 8,
    }
  }
  xl={
    Object {
      "span": 4,
    }
  }
  xs={
    Object {
      "span": 12,
    }
  }
>
  <Card
    actions={
      Array [
        <Icon
          onClick={[Function]}
          type="delete"
        />,
      ]
    }
    className="top-margin"
    hoverable={true}
    onClick={[Function]}
  >
    <Title
      level={4}
    />
    <Paragraph
      level={4}
    />
  </Card>
</Col>
`;
